cime_config/machines/config_batch.xml

```diff
@@ -550,7 +550,24 @@
      </queues>
    </batch_system>
 
-   <batch_system MACH="summit" type="lsf" >
+   <batch_system MACH="summit-cpu" type="lsf" >
```

Comment:
It appears to me that summit-gpu and summit-cpu have the same specs. Doesn't CIME support regex for selectors? If so, wouldn't `MACH="summit-(gpu|cpu)"` work, so that we avoid code repetition?

Reply:
not 100% sure about the regex in xml, let me test it to confirm

Reply:
regrex doesn't work in my test here

Reply:
Can you try summit.*

Reply:
yes, I tried, and it doesn't work as well

Reply:
bummer.

Reply:
Yep, no regex for MACH selectors